.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  lint-dockerfile:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Check Dockerfile compliance
+        run: docker run -i --rm hadolint/hadolint < Dockerfile
```

Comment:
@audrium What about adding `make lint` target as for the other repositories such as `r-e-jupyter`?